packages/@webex/plugin-meetings/src/locus-info/parser.ts

```diff
@@ -716,10 +723,16 @@ export default class Parser {
       // wait for desync response
       this.pause();
     } else if (lociComparison === USE_INCOMING) {
+      const locusUrlChanged = newLoci?.url !== this.workingCopy?.url;
+
       // update working copy for future comparisons.
       // Note: The working copy of parser gets updated in .onFullLocus()
       // and here when USE_INCOMING locus.
       this.workingCopy = newLoci;
+
+      if (this.workingCopy?.url && locusUrlChanged) {
```

Comment:
I think this will trigger a sync every time on the first delta event that we get, because `this.workingCopy` will be `null` when it's read in line 726 and that's not what we want, right?

Reply:
but it's set to `newLoci` just 2 lines above, so it will always be truthy, won't it?